### DISCUSSION TITLE
osu! disappearing from taskbar and task manager

### DISCUSSION THREAD

**Discussion by MeerukYang** [ppy/osu#29384] [Q&A]
### Type

Other

### Bug description

Using some certain program/software that access .lnk file to start osu! would make osu! process to disappear at taskbar; in this case it can be found and then closed by task manager.

![无标32423424题](https://github.com/user-attachments/assets/8af43d0d-6ad6-43c2-8bf8-1c926b27bdec)

But sometimes even task manager can't find osu! process, thus the bgm at menu would play forever. and unfortunately somestimes u can hear the notification sound, indicating that process is actually alive.
using taskkill command cannot kill the process in this case.

![无5675685698标题](https://github.com/user-attachments/assets/e7f948f5-3fc6-4e9c-bcea-e25b73e96b55)

both cannot display osu! window.

currently i am using [this web proxy software](https://www.liuxing.com/). (to get a better internet condition while playing osu). this is the software mentioned above. but I tried to start some other games with it by .lnk files and their icons and processes can normally showed up. so i assume it is osu!'s problem (sorry T^T).

### Version

2024.731.0

### Logs

restarted my PC since can't export logs, so they may differ.
[compressed-logs.zip](https://github.com/user-attachments/files/16571638/compressed-logs.zip)

**Comment by MeerukYang** [comment]
restarting explorer.exe couldn't help.

**Comment by MeerukYang** [comment]
the process can be found in some volume settings
![无标745756478578题](https://github.com/user-attachments/assets/47ccca42-c9aa-4886-94c3-7961c70be6c2)

**Comment by k3tzchen** [comment]
What about the Item-Tray? Back when I had Windows, osu! sometimes started hidden and was only accessible using the Icon in the Item-Tray (for you in the bottom right).

**Comment by MeerukYang** [comment]
> What about the Item-Tray? Back when I had Windows, osu! sometimes started hidden and was only accessible using the Icon in the Item-Tray (for you in the bottom right).

No, it didn't show up.
![无标342342342题](https://github.com/user-attachments/assets/429ededc-b4bb-4506-b165-2658f6639cfc)

**Comment by k3tzchen** [comment]
Never really used Command Prompt, but does it show any errors when you execute osu! using it?

**Comment by MeerukYang** [comment]
I've run `taskkill /IM osu!.exe` in cmd.exe while osu progress couldn't be found in task manager. it turns like this - this didn't run as administrator 
![无324324234423534题](https://github.com/user-attachments/assets/dcde1fe7-6712-4503-b261-e61f12930ded)
however when I run command above as adminstrator (twice), the process indeed shutdowned.

**Reply by k3tzchen** [reply]
I've more meant starting osu! using Prompt. So you can see if an error gets reported into the Prompt. 

Edit: This would also technically fix your "osu! process cannot be found" using taskkill. Since closing the Prompt Window also exits the osu! process (since its a child process of the Prompt)
